templates/CRM/Contribute/Form/AdditionalPayment.tpl

```diff
@@ -103,7 +103,7 @@
       </div>
```

Comment:
This closing div closes the crm-accordion-body too soon so the check field is outside the border. This `</div>` should be removed, and the one below (line 106) brought back.

<img width="533" alt="image" src="https://github.com/civicrm/civicrm-core/assets/1175967/ec3c598b-93e9-4cac-ba29-0749a2b8bf8f">

Reply:
I saw this one also but I think this should be a separate PR later since it existed before we did these changes.

I created a new ticket to address this at [issue#69](https://lab.civicrm.org/dev/user-interface/-/issues/69)